gtsfm/frontend/descriptor/convopt.py

```diff
@@ -0,0 +1,43 @@
+"""ConvOpt Descriptor implementation.
+
+The method was proposed in 'Learning Local Feature Descriptors Using Convex Optimisation' and is implemented by
+wrapping over OpenCV's API.
+
+Note: this is a standalone descriptor.
+
+References:
+https://www.robots.ox.ac.uk/~vedaldi/assets/pubs/simonyan14learning.pdf
+
+Authors: John Lambert
+"""
+import cv2 as cv2
```

Comment:
Unit tests for convopt missing. Actually, can you remove any new module addition from this PR?

Reply:
Sure, I'll remove here, and put in a separate PR.